plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -40,45 +40,55 @@ const waitForReceipt = async (whatLog, transaction) => {
 const setDepositVerifier = async (whatLog, ethVault, ethDepositVerifier, gnosisMultisigAbi, gnosisMultisigAddress, deployerAddress) => {
     const setDepositVerifierCall = web3.eth.abi.encodeFunctionCall(ethVault.abi.find(o => o.name === 'setDepositVerifier'), [ethDepositVerifier.address]);
     const gnosisSetDepositVerifier = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [ethVault.address, 0, setDepositVerifierCall]);
-    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier });
+    const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier }) * 2;
```

Comment:
nitpick: the `await web3.eth.estimateGas()` part is quite long, reading wise the `*2` is not that obvious at my first glance. Might be slightly clearer with the following

```js
const estimateGas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier })

const transaction = await web3.eth.sendTransaction({ gas: estimateGas * 2, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier });
```

Reply:
makes sense!

also,

all of this is really not optimal at the moment, but it onblocks me on some other things. I will come back to this and make some adjustments - (like to be able to retry a certain transaction by maintaining some sort of state (like truffle migration does)). just FYI.